index.html

```diff
@@ -1225,7 +1225,7 @@ <h4 class="subtitle endnote-subtitle">Books</h4>
 					<h4 class="subtitle endnote-subtitle">Articles</h4>
 					<ul>
 						<li>Митко Димитров: Ако на България ще ѝ стане по-добре, готов съм да върна парите (Mitko Dimitrov: If
-							Bulgaria is going to feel better, I will give up the money). www.btvnovinite.bg (Accessed 12 October 2019)
+							Bulgaria is going to feel better, I will give up the money).www.btvnovinite.bg (Accessed 12 October 2019)
```

Comment:
I don't see why these have changed?

Reply:
Hm, I think this is done automatically by VS Code, because of some unusual line terminators. If you want, I can revert them back to the original. 
![image](https://user-images.githubusercontent.com/63718744/221535822-d30fa826-1fa4-4abe-934f-4fc0e721749e.png)

Reply:
Ah, good point, leave the change then. Thanks for pointing it out!